Engine/DataFeeds/DataManager.cs

```diff
@@ -66,18 +66,26 @@ public DataManager(IDataFeed dataFeed, UniverseSelection universeSelection, IAlg
         #region IAlgorithmSubscriptionManager
 
         /// <summary>
-        ///     Flags the existence of custom data in the subscriptions
+        /// Flags the existence of custom data in the subscriptions
         /// </summary>
         public bool HasCustomData { get; set; }
 
         /// <summary>
-        ///     Gets all the current data config subscriptions that are being processed for the SubscriptionManager
+        /// Gets all the current data config subscriptions that are being processed for the SubscriptionManager
         /// </summary>
         public IEnumerable<SubscriptionDataConfig> SubscriptionManagerSubscriptions =>
             _subscriptionManagerSubscriptions.Select(x => x.Key);
 
         /// <summary>
-        ///     Gets existing or adds new <see cref="SubscriptionDataConfig" />
+        /// Gets a list of all registered <see cref="SubscriptionDataConfig"/> for a given <see cref="Symbol"/>
+        /// </summary>
+        public List<SubscriptionDataConfig> GetSubscriptionDataConfigs(Symbol symbol)
+        {
+            return SubscriptionManagerSubscriptions.Where(x => x.Symbol == symbol).ToList();
```

Comment:
just opinions :)

In general, I usually prefer to just return the enumerable as moves the `ToList()` decision to the caller. For example, if the caller immediately wants to group the result of this it will end up looping over the result set twice. No need to change anything here, and sometimes, it's really nice to return the already materialized list instead of an enumerable, which is really just a definition and not a collection. A really good example where you always want to return a list is if there's synchronization issues or if the underlying collection doesn't permit modification during enumeration. The underlying storage collection here, IIRC, is a `ConcurrentDictionary` so he doesn't really care either way.